src/components/totals/Totals.tsx

```diff
@@ -0,0 +1,199 @@
+
+      {!TotalsIsLoading && (
+        <div className="flex flex-col sm:grid grid-cols-3 gap-5 my-5 ">
+          <div className="flex flex-wrap justify-center col-start-2">
+            {Currencies &&
+              Object.entries(Currencies).map(([key, value]) => {
+                return (
+                  <Button
+                    key={`currency-btn-${key}`}
+                    value={value}
+                    type="secondary"
+                    className={'mx-0.5'}
+                    onClick={() => {
+                      setCurrency(Number(key));
+                    }}
+                  >
+                    {String(value)}
+                  </Button>
+                );
+              })}
+          </div>
+          <div className="flex justify-center sm:justify-start">
+            <Button
+              key={`day-btn`}
+              className={'mx-0.5'}
+              type="secondary"
+              onClick={() => {
+                setChartScale('day');
+              }}
+            >
+              Day
+            </Button>
+            <Button
+              key={`week-btn`}
+              className={'mx-0.5 '}
+              type="secondary"
+              onClick={() => {
+                setChartScale('week');
+              }}
+            >
+              Week
+            </Button>
+            <Button
+              key={`month-btn`}
+              className={'mx-0.5'}
+              type="secondary"
+              onClick={() => {
+                setChartScale('month');
+              }}
+            >
+              Month
+            </Button>
+          </div>
+          <div className="flex justify-center  sm:col-start-3 ">
+            <DropdownDateRangePicker
+              handleDateChange={handleDateChange}
+              start_date={body.start_date}
+              end_date={body.end_date}
+            ></DropdownDateRangePicker>
+          </div>
+          {TotalsData[Currency] && (
+            <>
+              <div>
+                {' '}
+                <Total
+                  Title="Total Revenue"
+                  Amount={TotalsData[Currency].revenue.paid_to_date}
+                  Currency={TotalsData[Currency].revenue.code}
+                ></Total>
+              </div>
+              <div>
+                {' '}
+                <Total
+                  Title="Total Expenses"
+                  Amount={TotalsData[Currency].expenses.amount}
+                  Currency={TotalsData[Currency].expenses.code}
+                ></Total>
+              </div>
+              <div>
+                {' '}
+                <Total
+                  Title="Outstanding"
+                  Amount={'70000.52'}
+                  Currency={TotalsData[Currency].revenue.code}
+                ></Total>
+              </div>
+            </>
+          )}
+        </div>
+      )}
+
+      {ChartDataIsLoading && <Spinner />}
+
+      <Card className="px-4 ">
```

Comment:
Class names on this shouldn't be needed as Container component has child spacing defined.